src/keycloak/keycloak_openid.py

```diff
@@ -689,7 +689,7 @@ def get_permissions(self, token, method_token_info="introspect", **kwargs):
         """
         if not self.authorization.policies:
             raise KeycloakAuthorizationConfigError(
-                "Keycloak settings not found. Load Authorization Keycloak settings ."
+                "Keycloak settings not found. Load Authorization Keycloak settings."
```

Comment:
I hope this won't be a breaking change :D
Can revert it if you want to play it safe